src/tests/stdlib/contains_multiple.ab

```diff
@@ -0,0 +1,6 @@
+import { contains_multiple } from "std/text"
```

Comment:
Please put blank line after `import` statements, here and in `src/tests/stdlib/extract.ab`.